release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -314,6 +314,17 @@ The cluster network can be expanded to support the addition of nodes to the clus
 
 In {product-title} {product-version}, {lvms} is supported in dual-stack for IPv4 and IPv6 network environments. For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network].
 
+[id="ocp-4-13-storage-lvms-resource-reduction"]
+==== Resource reduction of {lvms}
+
+In {product-title} {product-version}, the CPU and memory usage of {lvms} is reduced. The container image sizes are also decreased and you can apply labels to LVM {lvms} resources to enable workload partitioning.
```

Comment:
RN for [OCPVE-226](https://issues.redhat.com/browse/OCPVE-226)

@suleymanakbas91 Please take a look when you have the time.

Reply:
`you can apply labels to LVM {lvms} resources to enable workload partitioning.` The necessary labels are already applied to lvms resources so we could change this to something like `the required labels are applied to lvms resources so you can enable workload partitioning`.

Reply:
Sounds good :)